main.tf

```diff
@@ -49,18 +48,19 @@ resource "azurerm_lb_nat_rule" "azlb" {
 resource "azurerm_lb_probe" "azlb" {
   count               = length(var.lb_port)
   name                = element(keys(var.lb_port), count.index)
-  resource_group_name = azurerm_resource_group.azlb.name
+  resource_group_name = data.azurerm_resource_group.azlb.name
   loadbalancer_id     = azurerm_lb.azlb.id
   protocol            = element(var.lb_port[element(keys(var.lb_port), count.index)], 1)
   port                = element(var.lb_port[element(keys(var.lb_port), count.index)], 2)
   interval_in_seconds = var.lb_probe_interval
   number_of_probes    = var.lb_probe_unhealthy_threshold
+  request_path        = lower(element(var.lb_port[element(keys(var.lb_port), count.index)], 3)) == "http" || lower(element(var.lb_port[element(keys(var.lb_port), count.index)], 3)) == "https" ? element(var.lb_port[element(keys(var.lb_port), count.index)], 4) : ""
```

Comment:
In the [Azure LB TF doc](https://www.terraform.io/docs/providers/azurerm/r/lb_probe.html), the "request_path" refers to a URI. Seems the assigned value to request_path is not a URI type. Or did I misunderstand anything?

Reply:
Yes. It's defined as a URI. But there is no validation restriction in `lb_probe_resource` and in service api. I have tested the example works fine. I assume `\` is acceptable for the service.